src/Home/StartupIdea/StartupIdea.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import pic1 from '../../Images/scenes-03-2.png';
+
+const StartupIdea = () => (
+  <div className="startup-idea">
+    <div className="container">
+      <h1>Have a startup idea? Tell us more</h1>
+      <p>Feel free to ask for tips and guidance to validate and skyrocket your startup idea into reality</p>
+      <button className="btn startup-idea-button">LET US KNOW</button>
+      <img src={pic1} alt="pic" />
```

Comment:
```suggestion
      <img src={pic1} alt="ActualAltTag" />
```

Add more descriptive alt's, something like Startup Idea Photo etc

Reply:
Please use more declarative alt's throughout the website, it is very important for the SEO, as this is info website. 

https://www.mainstreetroi.com/alt-text-for-images-do-you-really-need-it/